src/codeu/chat/server/Server.java

```diff
@@ -83,6 +83,15 @@ public void onMessage(InputStream in, OutputStream out) throws IOException {
       }
     });
 
+    /*The purpose of this command is to pass the 'startTime' to the
+    client, when requested by him/her. */
+    this.commands.put(NetworkCode.SERVER_INFO_REQUEST, new Command() {
+      @Override
+      public void onMessage(InputStream in, OutputStream out) throws IOException {
+        Serializers.INTEGER.write(out, NetworkCode.SERVER_INFO_RESPONSE);
+        Time.SERIALIZER.write(out, view.getInfo().startTime);
```

Comment:
If you're not sure what I mean by the 'same key' try looking up hashmap or treemap to figure out how it works 
(this is where I learned) https://www.cs.utexas.edu/~scottm/cs314/handouts/slides/topic9Maps_4Up.pdf